docs/source/ngv.rst

```diff
@@ -0,0 +1,20 @@
+Neuro-Glia-Vasculature (NGV)
+============================
+
+.. image:: img/ngv.drawio.svg
+   :alt: NGV Diagram
+   :align: center
+
+The NGV simulation requires three node populations, each stored in separate files: **neocortex_neurons**, **astrocytes**, and **vasculature**.
+
+- **Neocortex_neurons** represent standard neocortex neurons.
+- **Astrocytes** represent glial cells.
+- The **vasculature** file is unique: although it is structured and treated as a node file by Neurodamus and the SONATA convention, each node (row in the table) actually refers to a vasculature segment. The vasculature itself is divided into morphology nodes (used internally for structure reconstruction), sections (optional), and segments. Currently, Neurodamus only reads the file and does not utilize this additional information. For more details, refer to the official documentation:  
```

Comment:
Do we need to mention that how we read the `start_diameter` and `end_diameter` and build the `sec(0.5).vascouplingB.R0pas`? https://github.com/openbraininstitute/neurodamus/blob/2942e785a280b0329f48dec2cceb5e86bcb4cc8a/neurodamus/ngv.py#L569

Reply:
I do not think so. It is already explained in the offical docs linked. Also, the implementation details should not go in the docs as things may change (without changing the API)

Reply:
ok, that's fine.